src/templates/_card.twig

```diff
@@ -0,0 +1,15 @@
+{# templates/_card.twig #}
+<div class="card">
+    <div class="card-header">
+        <p>{{ article.type }}</p>
+    </div>
+    <div>
+        <img src="{{ article.imagePath }}" alt="no-image" />
+    </div>
+    <div class="card-content">
+        <h3>{{ article.title }}</h3>
+        <h4>{{ article.author }}</h4>
+        <p> {{ article.description }}</p>
+        <a href="{{ article.link }}">See more</a>
```

Comment:
please remove any `article.` attribution or any hint about parent object
this card component should be semantically independent from any entities like "article" or "blog"
please use only variables like `{{ type }}`, `{{ title }}`, etc.
In order to correctly include this template, use this syntax:
```

{% include '_cards.twig' with {
	'title': article.title,
	'image': article.image,
} only %}```
don't forget to check out the `only` directive